packages/core/src/core/contentGenerator.test.ts

```
@@ -47,4 +51,73 @@ describe('contentGenerator', () => {
+      authType: AuthType.USE_OPENAI,
+    });
+    
+    expect(OpenAIContentGenerator).toHaveBeenCalledWith(
+      'test-api-key',
+      'https://custom.openai.com/v1',
+      'gpt-4',
+    );
+    expect(generator).toBe(mockGenerator);
+    
+    // Clean up
+    delete process.env.OPENAI_BASE_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)